modules/rostests/winetests/bcrypt/bcrypt.c

```diff
@@ -760,10 +764,41 @@ START_TEST(bcrypt)
 {
     HMODULE module;
 
+#ifdef __REACTOS__
+    HMODULE hAdvapi32Dll;
+
+    // bcrypt is available on WVista (and ReactOS), not on WS03.
+    if (!LoadLibraryW(L"bcrypt.dll"))
+    {
+       win_skip("LoadLibrary(\"bcrypt.dll\") failed. This is expected on WS03\n");
+       return;
+    }
+
+    // ToDo: Use __HrLoadAllImportsForDll("bcrypt.dll") after CORE-10957 is fixed.
+
+    // advapi32/RegGetValueW() is available on WXP-Pro-x64 and WS03sp1, not on WXP.
+    hAdvapi32Dll = LoadLibraryW(L"advapi32.dll");
+    pRegGetValueW = hAdvapi32Dll ? (PVOID)GetProcAddress(hAdvapi32Dll, "RegGetValueW") : NULL;
+#endif
+
     module = GetModuleHandleA( "bcrypt.dll" );
 
     test_BCryptGenRandom();
+
+#ifdef __REACTOS__
+    if (!pRegGetValueW)
```

Comment:
Just use the upstream fix: https://source.winehq.org/git/wine.git/commitdiff/b5e5be13d20adff5d7b784e71c7a48fb95fde2c8

Reply:
Indeed! See further details/questions in [ROSTESTS-296 comment](https://jira.reactos.org/browse/ROSTESTS-296?focusedCommentId=100169&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-100169).

Reply:
Then, let's wait for #208.

Reply:
Eventually superseded by #208.